package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@oasisprotocol/ledger",
-  "version": "0.2.3",
+  "version": "0.2.4",
```

Comment:
To keep consistency with the new methods, **getAddressAndPubKey** was renamed to **getAddressAndPubKey_ed25519** (new method is called **getAddressAndPubKey_secp256k1**)
To respect SemVer, major field should be bumped. However, if you don't want to make breaking changes in the API, it's possible to keep the old names.
I think that this should be decided by Oasis team 😄

Reply:
Good point! I will bump the major so that we have a version that respects SemVer, since the changes in this PR ask for that. Then Oasis Team can decide to merge this as is or we change the old method names.